src/Api/IriConverterInterface.php

```diff
@@ -13,78 +13,15 @@
 
 namespace ApiPlatform\Core\Api;
 
-use ApiPlatform\Core\Exception\InvalidArgumentException;
-use ApiPlatform\Core\Exception\ItemNotFoundException;
-use ApiPlatform\Core\Exception\RuntimeException;
 
 /**
  * Converts item and resources to IRI and vice versa.
  *
+ * @deprecated please use ItemToIriConverterInterface or IriToItemConverterInterface to fit your needs instead
```

Comment:
one @trigger_errors should be added.
```
IriConverterInterface is deprecated since API Platform 2.3 and will be removed in API Platform 3.0, use ItemToIriConverterInterface and/or IriToItemConverterInterface instead.
```
both class should be fully qualified.